xkcd.py

```diff
@@ -0,0 +1,36 @@
+import re
+import requests
+from bs4 import BeautifulSoup
+import random
+import os
+
+
+l=[] #For storing random numbers
```

Comment:
Why to declare it so earlier ? it should be near the code it is being used in.
Actually I am not gonna use it in the code(read my next comment for followup). Just telling to declare variables close to the code block they are used in, unless they are used at many places.